import_busstops.py

```diff
@@ -7,6 +7,7 @@ class BusImportCommand(CSVImportCommand):
 
     def __init__(self, *kargs, **kwargs):
         self.dry_run = kwargs.pop('dry_run')
+        kwargs['num_expected_records'] = 430525
```

Comment:
ok, but we should pass it as parameter, we don't know if next time we download the Naptan zip files that number will be still the same. Train stations don't change very often but bus stops do, as far as I know